File: src/main/java/com/jdc/db/jpa/query/FirebirdQueryBuilder.java
/*
 * FirebirdQueryBuilder.java
 *
 * Copyright 2008 Jeff Campbell. All rights reserved. Unauthorized reproduction 
 * is a violation of applicable law. This material contains certain 
 * confidential or proprietary information and trade secrets of Jeff Campbell.
 */
package com.jdc.db.jpa.query;

import com.jdc.db.shared.query.FirebirdUtil;

/**
 *
 * @author Jeff
 */
public class FirebirdQueryBuilder extends JPAQueryBuilder {

    @Override
    public String formatIgnoreCaseLikeClause(String column, String value) {
        return FirebirdUtil.formatIgnoreCaseLikeClause(column, value);
    }
}

File: dbtools-query/src/commonMain/kotlin/org/dbtools/query/sql/FirebirdQueryBuilder.kt
/*
 * FirebirdQueryBuilder.java
 *
 * Created on October 27, 2007
 *
 * Copyright 2007 Jeff Campbell. All rights reserved. Unauthorized reproduction 
 * is a violation of applicable law. This material contains certain 
 * confidential or proprietary information and trade secrets of Jeff Campbell.
 */
package org.dbtools.query.sql

import org.dbtools.query.shared.util.FirebirdUtil

/**
 *
 * @author Jeff
 */
class FirebirdQueryBuilder : SQLQueryBuilder() {
    override fun formatIgnoreCaseLikeClause(field: String, value: String): String {
        return FirebirdUtil.formatIgnoreCaseLikeClause(field, value)
    }
}
